.changeset/nice-schools-remember.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-adr': patch
```

Comment:
Just realized this should be `minor` since we're changing the `AdrClient` interface

Reply:
Alright, I've update the changeset of #17215 as well to match the change :+1: